Source/Data/bin/buildFreq.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Maybe we need to update the README file to notify the developers that they need Python 3 installed

Reply:
Xcode command line tools shipped by the required Xcode may already have Python3 installed, I guess·

Still, it worths mentioning.